src/main/java/org/jenkinsci/test/acceptance/docker/Docker.java

```diff
@@ -27,6 +27,7 @@
  * Use this subsystem by injecting this class into your test.
  *
  * @author Kohsuke Kawaguchi
+ * @author asotobueno
```

Comment:
BTW you can also feel free (IMO) to just delete any `@author` tags. Anyone who cares can `git blame` for themselves.